proms-resources/frontend/src/main/frontend/src/TOU.json

```
@@ -1,4 +1,80 @@
+  "text" : [
+    "These terms and conditions (the \"Agreement\") for a symptoms survey located at ePRO.uhn.ca and any associated website, portal, or similar service (collectively, the \"Application\"), constitute a legal agreement and is entered into by and between you (\"you\", \"your\") and the University Health Network (\"UHN\", \"we\", \"us\", \"our\").\\\n",
+    "**IMPORTANT! BY USING OR ACCESSING THE APPLICATION, YOU AGREE TO BECOME BOUND BY THE AGREEMENT. IF YOU DO NOT AGREE WITH THE AGREEMENT, YOU MUST NOT USE OR ACCESS THE APPLICATION. NOTE THAT THERE ARE ALTERNATIVE METHODS TO COMPLETE THE SURVEY IF YOU WOULD NOT LIKE TO PROCEED WITH THIS APPLICATION.**\\\n",
+    "This Agreement governs your access to the Application and use, including any content, functionality, and services offered on or through the Application.\\\n",
+    "The Application is intended to act as an information collection tool for physicians to better prepare for appointments with patients and to improve record keeping. The Application is intended to facilitate symptom reporting to healthcare providers, for quality monitoring and improvement, and general patient education relative to cancer symptom status. It is not a medical device. Nothing contained in the Application should be construed, directly or indirectly, as the practice of medicine or providing medical advice or medical services by UHN. The Application is not intended to diagnose, cure, mitigate, treat, or prevent disease or other conditions.\\\n",
+    "####Not Medical Advice.\\\n",
```

The headings don't get rendered because the correct syntax is `#### Title`, with space between the heading marker `####` and the text.
```suggestion
    "#### Not Medical Advice.\\\n",
```

---

Fixed in 8c43d6e